addons/survey/static/src/scss/survey_views.scss

```diff
@@ -1,6 +1,45 @@
 .o_kanban_card_survey {
+    min-height: 100px;
+    
     .certification_icon {
-        font-size: 20px;
+        overflow: hidden;
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        height: 100%;
+        width: 100%;       
+
+        .fa-trophy{
+            position: absolute;
+            font-size: 105px;
+            left: -45px;
+            bottom: 10px;
+            opacity: 0.15;
+        }
+    }
+
+    .fa-clock-o {
+        visibility: hidden;
+        
+        @include media-breakpoint-down(sm) {
+            visibility: visible;
+        }
+    }
+
+    .o_activity_color_planned,
+    .o_activity_color_today,
+    .o_activity_color_overdue{
+        visibility: visible;
+    }
```

Comment:
you can remove this and set o_activity_color_default to hidden by default and visible on hover (instead of fa-clock-o)